tempodb/encoding/vparquet2/block_traceql.go

```diff
@@ -726,6 +729,16 @@ func createAllIterator(ctx context.Context, primaryIter parquetquery.Iterator, c
 		traceConditions    []traceql.Condition
 	)
 	for _, cond := range conds {
+		// check for existence filter {.a != nil}
+		// and change it to opnone
```

Comment:
I was thinking we could do this in the engine around [here](https://github.com/grafana/tempo/blob/main/pkg/traceql/ast_conditions.go#L17) and then maybe it would work without storage-layer changes.  What do you think?